src/execution/physical_plan/plan_window.cpp

```diff
@@ -1,14 +1,36 @@
+#include "duckdb/execution/operator/aggregate/physical_streaming_window.hpp"
 #include "duckdb/execution/operator/aggregate/physical_window.hpp"
 #include "duckdb/execution/operator/projection/physical_projection.hpp"
 #include "duckdb/execution/physical_plan_generator.hpp"
-#include "duckdb/planner/operator/logical_window.hpp"
 #include "duckdb/planner/expression/bound_reference_expression.hpp"
 #include "duckdb/planner/expression/bound_window_expression.hpp"
+#include "duckdb/planner/operator/logical_window.hpp"
 
 #include <numeric>
 
 namespace duckdb {
 
+bool IsStreaming(vector<unique_ptr<Expression>> &select_list) {
```

Comment:
To avoid amalgamation problems, should this be a) static and b) have a more descriptive name like `IsStreamingWindow`?

Reply:
Good idea!